src/docs/truffle/how-to/use-the-build-pipeline.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Using the Build Pipeline
+title: Use the Build pipeline
 layout: docs.hbs
 ---
-# Using the Build Pipeline
+# Use the Build pipeline
```

Comment:
Again (minor nipick) a discrepancy between the casing on the left-navigation ("Use the build pipeline") and the page title ("Use the Build pipeline")